docs/install/README.md

```diff
@@ -80,15 +80,14 @@ Don't worry if you aren't sure of these items yet; you can always make changes t
 Using the included Docker utility script, updating is as simple as running:
 
 ```bash
-./docker.sh update-self
-./docker.sh update
+./docker.sh update-self && echo "y" | ./docker.sh update
```

Comment:
I'd probably not just replace the existing "manual" command but add this as a second more "automatic" way to update.

Something like this:

```bash
./docker.sh update-self && ./docker.sh update
```
```
Using the following command to update will automatically answer yes to the question about overwriting the docker-compose.yml:
```

```bash
./docker.sh update-self && echo "y" | ./docker.sh update
```

Reply:
Of course, you are correct. thanks! I will add it again.